cmd/create/operatorroles/cmd.go

```diff
@@ -181,7 +181,6 @@ func run(cmd *cobra.Command, argv []string) {
 	var isHcpSharedVpc bool
 	var err error
 	if !args.hostedCp {
-		rosa.HostedClusterOnlyFlag(r, cmd, hostedZoneRoleArnFlag)
```

Comment:
It should include the other hosted zone though, right? The hcp internal comms

Reply:
ahh my bad, thanks, Monday brain scrambles